frontend/src/components/pages/GlobalForms/WaiversTab.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ClauseInfo } from "../../../types/AdminTypes";
+import WaiverSectionCard from "./WaiverSectionCard";
+  
+  interface WaiversTabProps {
+    clauses: Array<ClauseInfo>;
+  }
+  const WaiversTab = ({
+    clauses
+  }: WaiversTabProps): JSX.Element => {
+
+    function intToChar(number: number) {
+        const code = 'A'.charCodeAt(0);  
+        return String.fromCharCode(code + number);
+    }
+    return (
+        <>
+        {clauses.length && (clauses).map((clause, idx)=> {
+            return <WaiverSectionCard key={idx} title={`Section ${intToChar(idx)}`}
```

Comment:
Can you rename the key to  " (backtick) waiver_section_card_${index} (backtick) ". this makes the key more meaningful instead of just a number. 

Note (backtick) represents a ` . Github kept changing that to think of the text as code